diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1f18782f3..e6a6e7d2c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -536,6 +536,17 @@ if(NOT ${CMAKE_VERSION} VERSION_LESS "3.2")
   set(MAYBE_USES_TERMINAL USES_TERMINAL)
 endif()
 
+FILE(WRITE "${CMAKE_CURRENT_BINARY_DIR}/boringssl-config.cmake" "include(\"\${CMAKE_CURRENT_LIST_DIR}/boringssl-crypto-targets.cmake\")\ninclude(\"\${CMAKE_CURRENT_LIST_DIR}/boringssl-ssl-targets.cmake\")")
+
+INSTALL(TARGETS crypto EXPORT boringssl-crypto-targets
+        INCLUDES DESTINATION include)
+INSTALL(TARGETS ssl EXPORT boringssl-ssl-targets
+        INCLUDES DESTINATION include)
+INSTALL(EXPORT boringssl-crypto-targets DESTINATION cmake NAMESPACE boringssl:: COMPONENT crypto)
+INSTALL(EXPORT boringssl-ssl-targets DESTINATION cmake NAMESPACE boringssl:: COMPONENT ssl)
+INSTALL(DIRECTORY ${CMAKE_SOURCE_DIR}/include/ DESTINATION include)
+INSTALL(FILES "${CMAKE_CURRENT_BINARY_DIR}/boringssl-config.cmake" DESTINATION cmake)
+
 if(UNIX AND NOT APPLE AND NOT ANDROID)
   set(HANDSHAKER_ARGS "-handshaker-path" $<TARGET_FILE:handshaker>)
 endif()
