diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -640,3 +640,21 @@ add_custom_target(
     WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
     DEPENDS all_tests bssl_shim handshaker fips_specific_tests_if_any
     USES_TERMINAL)
+
+
+include(CMakePackageConfigHelpers)
+include(GNUInstallDirs)
+
+configure_package_config_file(
+  "${CMAKE_SOURCE_DIR}/boringssl-config.cmake.in"
+  "${CMAKE_BINARY_DIR}/boringssl-config.cmake"
+  INSTALL_DESTINATION "${CMAKE_INSTALL_DATADIR}/boringssl"
+)
+install(FILES
+  "${CMAKE_BINARY_DIR}/boringssl-config.cmake"
+  DESTINATION "${CMAKE_INSTALL_DATADIR}/boringssl"
+)
+
+INSTALL(TARGETS crypto ssl EXPORT boringssl-targets)
+INSTALL(EXPORT boringssl-targets DESTINATION "${CMAKE_INSTALL_DATADIR}/boringssl" NAMESPACE boringssl::)
+INSTALL(DIRECTORY ${CMAKE_SOURCE_DIR}/include/ TYPE INCLUDE)
diff --git a/boringssl-config.cmake.in b/boringssl-config.cmake.in
new file mode 100644
--- /dev/null
+++ b/boringssl-config.cmake.in
@@ -0,0 +1,5 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/boringssl-targets.cmake")
+
+check_required_components(boringssl)
