diff --git a/CMakeLists.txt b/CMakeLists.txt
index a4a2ade..6a305bc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -72,7 +72,13 @@ else()
 endif()
 
 add_library(status-code INTERFACE)
-target_compile_features(status-code INTERFACE cxx_std_11)
+if (STATUS_CODE_REQUIRE_CXX20)
+  target_compile_features(status-code INTERFACE cxx_std_20)
+elseif(STATUS_CODE_REQUIRE_CXX14)
+  target_compile_features(status-code INTERFACE cxx_std_14)
+else()
+  target_compile_features(status-code INTERFACE cxx_std_11)
+endif()
 target_include_directories(status-code INTERFACE
   "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
   "$<INSTALL_INTERFACE:include>"
