diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4293966..6dd9a07 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -140,6 +140,14 @@ endif()
 # Set the standard definitions for these libraries and bring in the all_* helper functions
 include(QuickCppLibApplyDefaultDefinitions)
 
+if (DEFINED CMAKE_MSVC_RUNTIME_LIBRARY)
+  if (CMAKE_MSVC_RUNTIME_LIBRARY MATCHES "^MultiThreaded.*DLL$")
+    set_property(GLOBAL PROPERTY CXX_STATIC_RUNTIME OFF)
+  else()
+    set_property(GLOBAL PROPERTY CXX_STATIC_RUNTIME ON)
+  endif()
+endif()
+
 # Set the C++ features this library requires
 all_compile_features(PUBLIC
   # cxx_exceptions                        ## Annoyingly not supported by cmake 3.6
