diff --git a/include/llfio/v2.0/detail/impl/traverse.ipp b/include/llfio/v2.0/detail/impl/traverse.ipp
index 06354fe8..de2bcb0f 100644
--- a/include/llfio/v2.0/detail/impl/traverse.ipp
+++ b/include/llfio/v2.0/detail/impl/traverse.ipp
@@ -162,7 +162,7 @@ namespace algorithm
                 using_sso = true;
               }
             }
-#if _MSC_VER < 1933  // <= VS2022.2
+#if _MSC_VER /*< 1933*/  // <= VS2022.2
             // MSVC's list::splice() always copies :(
             workitem(const workitem &o) noexcept
                 : workitem(const_cast<workitem &&>(std::move(o)))
